apps/extension/src/content-scripts/injected-penumbra-global.ts

```diff
@@ -87,7 +87,7 @@ const postRequest = () => {
       window.origin,
     );
     request.promise
-      .catch(e => connection.reject(e))
+      .catch((e: unknown) => connection.reject(e))
```

Comment:
New eslint rule: https://typescript-eslint.io/rules/use-unknown-in-catch-callback-variable/

Catch needs to specify that it's an unknown